bangazonapi/views/product.py

```diff
@@ -266,7 +266,7 @@ def list(self, request):
 
         if number_sold is not None:
             def sold_filter(product):
-                if product.number_sold <= int(number_sold):
+                if product.number_sold >= int(number_sold):
```

Comment:
what if the price has cents?

Reply:
I'll need you to elaborate.  The ticket is in reference to the number of sold items, not the price.  Is there a part of the ticket i'm not understanding or another ticket this in reference too?  I just need clarification on what you're asking for specifically.

Reply:
wrong comment, sorry 😓 

there's a different ticket that has a very similar fix to this one. got them mixed up by not reading closely enough.

looks good. approving!